benchmark_utils/tv_numba.py

```diff
@@ -399,3 +398,37 @@ def gtv_mm_tol2(y, lmbd, K=1, max_iter=100, tol=1e-3):
         else:
             cost_prev = cost
     return x
+
+
+gtv_mm_tol2.jitter = nb.njit(
+    [
+        nb.types.Array(nb.types.float64, 1, "A")(
+            nbr1d64,
+            nb.types.float64,
+            nb.types.int16,
+            nb.types.int16,
+            nb.types.float64,
+        ),
+        nb.types.Array(nb.types.float32, 1, "A")(
+            nbr1d32,
+            nb.types.float32,
+            nb.types.int16,
+            nb.types.int16,
+            nb.types.float32,
+        ),
+    ],
+    error_model="numpy",
+)
+
+JITTED = False
+
+
+def jit_module():
```

Comment:
Why do you do this this way and not with the decorator? because the compile team is too long?

Reply:
indeed, the idea is do the jit-compilation only when we are going to use the module.